docs/reference_guides/model_libraries/generic/costing/process_costing.rst

```diff
@@ -0,0 +1,865 @@
+    m.fs.unit.tube_inlet.enth_mol[0].fix(4000)
+    m.fs.unit.tube_inlet.pressure[0].fix(101325.0)
+    
+    m.fs.unit.area.fix(1000)  # m2
+    m.fs.unit.overall_heat_transfer_coefficient.fix(100)  # W/m2K
+    
+    m.fs.costing = SSLWCosting()
+    m.fs.H101.costing = UnitModelCostingBlock(
+        default={
+            "flowsheet_costing_block": m.fs.costing,
+            "costing_method": SSLWCostingData.cost_fired_heater,
+            "costing_method_arguments": {
+                    "material_type": HeaterMaterial.CarbonSteel,
+                    "heat_source": HeaterSource.Fuel
+            }
+        }
+    )
+
+    m.fs.unit.initialize()
+
+    opt = SolverFactory('ipopt')
+    opt.options = {'tol': 1e-6, 'max_iter': 50}
+    results = opt.solve(m, tee=True)
+
+Units
+^^^^^
+
+It is important to highlight that the costing method interrogates the property 
+package to determine the units of this model, if the user provided the correct 
+units in the metadata dictionary (see property models for additional information), 
+the model units will be converted to the right units. 
+For example: in this example area is in m^2, while the cost correlations for heat 
+exchangers require units to be in ft^2. Therefore, the costing method will convert 
+the units to ft^2. The use of Pyomo-unit conversion tools is under development.
+
+Available IDAES Costing Module Methods
+--------------------------------------
+
+A default costing module has been developed primarily based on base cost and purchase cost correlations 
+from the following reference with some exceptions (noted in the documentation as appropiate).
+
+Process and Product Design Principles: Synthesis, Analysis, and Evaluation. Seider, Seader, Lewin, Windagdo, 3rd Ed. John Wiley and Sons. Chapter 22. Cost Accounting and Capital Cost Estimation
+
+Users should refer to the reference above for details of the costing correlations, however, a summary of this methods is provided below.
+    
+Table 2. Cost basis for each unit model.
+
+=========================== =========================  ===========
+Unit Model                  Basis                      Units      
+=========================== =========================  ===========
+heat exchanger              :math:`area`               ft^2       
+pump                        :math:`fluid_{work}`       ft^3/s     
+compressor                  :math:`mechanical_{work}`  hp         
+turbine                     :math:`mechanical_{work}`  hp         
+vessels                     :math:`D , L`              ft         
+fired heaters               :math:`heat\_duty`         BTU/hr         
+=========================== =========================  ===========
+
+Unit Mapping
+^^^^^^^^^^^^
+
+As a convenience for users, common unit models are mapped with appropriate costing methods. Therefore, for quick costing with defaults, users may import the `unit_mapping` dictionary and pass the unit model type as below:
+
+.. code:: python
```

Comment:
1. I would show this as a table so it is easily readable.
2. I do not believe there is any need to import this separately - it is part of the costing package itself.